gflownet.py

```diff
@@ -353,7 +372,7 @@ def get_training_data(self, batch_size):
 
         while envs:
             envs, actions, valids = self.forward_sample(
-                envs, policy="mixt", temperature=self.temperature
+                envs, policy="model", temperature=self.temperature
```

Comment:
The policy could be a parameter of the config, but I would suggest to keep it as "mixt" by default  ... we want to sample calling the model or chosing a random action among those possible sometimes in the general case

Reply:
I was comparing the results of experiments on main-new-al with previous code. In previous code, for the best performance on grid the forward policy was `model` -- that's why i changed that here for the time being (plus so that you could simply take this branch and run the experiments if you wanted to compare for yourself)